tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = unittests
```

Comment:
should be tested on all supported python (3.8 and up)

Reply:
done; let me know if its ok